src/components/UploadingJobsDisplay/components/FileItem/FileItem.js

```diff
@@ -23,23 +97,34 @@ class FileItem extends PureComponent {
 
   render() {
     const {
-      name,
       uploadedValue,
       size,
+      uploadStatus,
+      name,
     } = this.props;
 
+    const meta = getFileItemMeta({
+      uploadStatus,
+      name,
+    });
+
     return (
-      <div className={css.fileItem}>
-        <span>{name}</span>
-        <Progress
-          payload={this.progressPayload}
-          progressInfoType="messagedPercentage"
-          progressClassName={css.progress}
-          progressWrapperClassName={css.progressWrapper}
-          progressInfoClassName={css.progressInfo}
-          total={size}
-          current={uploadedValue}
-        />
+      <div className={classNames(css.fileItem, meta.fileWrapperClassName)}>
+        <div className={css.fileItemHeader}>
+          {meta.renderHeading()}
+        </div>
+
+        {meta.showProgress &&
+          <Progress
```

Comment:
Consider wrapping it with parenthesis as it is a multiline JSX expression.

Reply:
Agree as long as we act consistent with this. Should be okay now.